include/utils/math/geometry/pose2d.h

```diff
@@ -0,0 +1,197 @@
+#pragma once
+
```

Comment:
would it be nicer to put lines like 3-7 in their own header under like core/include/EigenDenseAdaptor

and that was you just include that one file rather than this whole block each time

Reply:
that would in fact be nicer yes that's the smart way

Reply:
oh actually idk why i didn't think of this before but you can undef with compiler flags. we could probably just do that with the link to the vex forum in the makefile